docs/cdn.md

```diff
@@ -18,28 +18,28 @@ Alternatively, use [compressed files](#compressed-file).
 
 ```html
 <!-- load css -->
-<link rel="stylesheet" href="//unpkg.com/docsify@2.0.0/themes/vue.css">
+<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@2.0.0/themes/vue.css">
```

Comment:
```suggestion
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4.10.2/themes/vue.css">
```

Mention the latest version, do the same for other specific version as well

Reply:
amended code based on review